manual/README.md

```diff
@@ -0,0 +1,15 @@
+# QMCPACK Manual
+
+This directory contains the LaTeX source for the QMCPACK manual. The
+PDF manual is not currently created by the build process and must be
+created manually. Note that due to the use of the bibtopic package to
+support two bibliographies, bibtex must be invoked twice:
+
+```
+pdflatex qmcpack_manual.tex
```

Comment:
Could this sequence of commands also be put into a shell script ('build_manual.sh')?

Reply:
Added.